lib/cc/engine/analyzers/python/main.rb

```diff
@@ -9,7 +9,8 @@ module Engine
     module Analyzers
       module Python
         class Main
-          DEFAULT_MASS_THRESHOLD = 50
+          DEFAULT_MASS_THRESHOLD = 40
+          BASE_POINTS = 1000
```

Comment:
The s-expressions generated from Python's AST is much more verbose than Ruby's which means the points have to be lower.